src/components/GameCard.js

```diff
@@ -1,68 +1,132 @@
- return(
-    <Card>
-        <Location>{location}</Location>
-        <Date>{date}</Date>
-       
-            <Headline>Player(s)</Headline>
-            <DataPlayers>{players}</DataPlayers>
-      
-            <Headline>Winner(s)</Headline>
-            <DataWinner>{winner}</DataWinner>
-        
-       
-            <Headline>Total Shots</Headline>
-            <DataShots>{shots}</DataShots>
-    </Card>
- )
+export default function GameCard({location, date, players, winner, shots, id, onDelete}) {
+ 
+    const [isSetToDelete, setIsSetToDelete] = useState(false)
+
+    return(
+        <Card>
+            {!isSetToDelete && (
+                <SavedGameContent>
+                    <Location>{location}</Location>
+                    <Date>{date}</Date>
+                    <PlayerWrapper>
+                        <Headline>Player(s)</Headline>
+                        <span>{players}</span>
+                    </PlayerWrapper>
+                    <WinnerWrapper>
+                        <Headline>Winner(s)</Headline>
+                        <span>{winner}</span>
+                    </WinnerWrapper>
+                    <ShotsWrapper>
+                        <Headline>Total Shots</Headline>
+                        <span>{shots}</span>
+                    </ShotsWrapper>
+                    <ButtonDeleteFirst onClick={() => setIsSetToDelete(true)}><BinIcon/></ButtonDeleteFirst>
+                </SavedGameContent>
+            )}
+            {isSetToDelete && (
+                <DeleteField>
+                    <span>Do you want to delete this gamecard?</span>
+                    <ButtonDeleteSecond onClick={() => onDelete(id)}><BinIcon/>Delete</ButtonDeleteSecond>
+                    <ButtonCancel onClick={() => setIsSetToDelete(false)}> X Cancel</ButtonCancel>
+                </DeleteField>
+            )}
+        </Card>
+    )
 }
 
 const Card = styled.section`
     padding: 10px;
-    display: grid;
-    grid-template-rows: repeat(3, 1fr);
-    grid-template-columns: repeat(3, 1fr);
-    grid-gap: 10px;
-    justify-items: center;
     box-shadow: 0 0 10px var(--primary-medium);
     border-radius: 10px;
     background-color: var(--primary-medium);
     font-size: 0.9rem;
-    text-align: center;
+`
+const SavedGameContent = styled.div`
+    display: grid;
+    grid-template-rows: repeat(4, auto);
+    grid-template-columns: repeat(3, 1fr);
+    grid-gap: 10px;
+    justify-items: left;
+`
+const Date = styled.div`
+    grid-column: 1;
+    grid-row-start: 1;
 `
 const Location = styled.div`
     grid-column: 1 / 3;
-    justify-self: start;
+    grid-row-start: 2; 
+    text-transform: uppercase;
 `
-const Date = styled.div`
-    justify-self: end;
+const PlayerWrapper = styled.div`
+    grid-column: 1/4;
+    grid-row-start: 3;
+`
+const WinnerWrapper = styled.div`
+    grid-column: 1;
+    grid-row-start: 4;
+`
+const ShotsWrapper = styled.div`
+    grid-column: 2;
+    grid-row-start: 4;
 `
```

Comment:
Nice! You used the entire selection of grid options! I learned / was reminded of a lot going through your code here